src/components/AnnouncementCarousel/AnnouncementCarousel.tsx

```diff
@@ -21,15 +29,15 @@ const CustomEllipse = ({ onClick }: any) => {
   )
 }
 
-const NextArrow = ({ onClick }: any) => {
+const NextArrow = ({ onClick }: CustomArrowProps) => {
```

Comment:
A true hero for all of these `any` remediations!

Reply:
No kidding, the lint output is so clean now 

![SCR-20231012-n2c](https://github.com/USSF-ORBIT/ussf-portal-client/assets/940173/911bb96b-7598-4086-a91b-81ce8d11f565)